lib/class-wp-rest-site-controller.php

```diff
@@ -88,10 +88,164 @@ public function get_item_schema() {
 			'title'      => 'site',
 			'type'       => 'object',
 			'properties' => array(
-				'title' => array(
-					'description' => __( 'Site Title' ),
+
+				// Reference: https://docs.google.com/spreadsheets/d/1vI-s8MjbEllCR0_0BmBgaGvKNm1Fa42MCoqaKCwbRyQ/edit#gid=0
+
+				// Discussion
+
+				'avatar_default' => array(
+					'description' => __( 'Default Avatar' ),
+					'type'        => 'string',
+					'screen'      => 'discussion',
+					'context'     => array( 'view', 'edit' ),
+					'public'      => true,
```

Comment:
I agree that maybe the screen attribute is not that useful but since some options are private and others are public we will need a mapping system to determine what options can be retrieved as an unauthenticated user...right?